website/docs/support/support.md

```diff
@@ -0,0 +1,38 @@
+---
+id: support
+title: Support
+---
+
+## How to create an issue
+
+We will show you how to create an issue with farm-ng.
+
+Issue are created for bugs, feature requests, documentation requests.
+
+To create an issue for documentation requests, please go [here](https://github.com/farm-ng/amiga-dev-kit/issues/new/choose)
+
+![doc req](https://user-images.githubusercontent.com/64480560/209391577-688b70ea-edae-4780-8c52-d5aa5c6854ff.gif)
+
+To create an issue for documentation issues, please go [here](https://github.com/farm-ng/amiga-dev-kit/issues/new/choose)
+
+![doc issue](https://user-images.githubusercontent.com/64480560/209391689-090f4ce7-fad2-4359-ac71-c0efb36dc0e6.gif)
+
+
```

Comment:
Keep it consistent for dev-kit & amiga-brain since both have 1 link & 2 gifs

Reply:
can you please explain this more. What do you mean?

Reply:
Sure, see new comment